ansible/roles/grafana/defaults/main.yml

```diff
@@ -0,0 +1,182 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+# debops.grafana default variables
+# ================================
+
+# .. contents:: Sections
+#    :local:
+#
+# .. include:: ../../../../includes/global.rst
+
+
+# Domain name configuration [[[
+# -----------------------------
+
+# .. envvar:: grafana__hostname [[[
+#
+# The hostname part of the Grafana installation
+grafana__hostname: 'grafana'
+                                                                   # ]]]
+
+# .. envvar:: grafana__fqdn [[[
+#
+# String  of the Fully Qualified domain names on which the Grafana
+# application will be available, used by the webserver. It will be
+# used as the e-mail sender domain name.
+grafana__fqdn: '{{ grafana__hostname }}.{{ grafana__domain }}'
+
+                                                                   # ]]]
+# .. envvar:: grafana__domain [[[
+#
+# The DNS domain used by other variables in the ``debops.grafana`` role.
+grafana__domain: '{{ ansible_local.core.domain
+                    if (ansible_local|d() and ansible_local.core|d() and
+                        ansible_local.core.domain|d())
+                    else ansible_domain }}'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# Grafana configuration [[[
+# -------------------------
+
+# .. envvar:: grafana__nginx_enabled: True [[[
+#
+# Enable or disable Nginx dependencies.
+grafana__nginx_enabled: True
+
+                                                                   # ]]]
+# .. envvar:: grafana__upstream_apt_id [[[
+#
+# The Grafana fingerprint.
+grafana__upstream_apt_id: '4E40 DDF6 D76E 284A 4A67  80E4 8C8C 34C5 2409 8CB6'
+
+                                                                   # ]]]
+# .. envvar:: grafana__upstream_apt_repository [[[
+#
+# The APT repository URI of the upstream Grafana repository.
+grafana__upstream_apt_repository: 'deb https://packages.grafana.com/oss/deb stable main'
+
```

Comment:
Can you remove the empty line here? That way the variables are evenly spaced from each other. This is commonly done at the end of the sections.

Reply:
removed